saleor/warehouse/availability.py

```diff
@@ -232,17 +256,10 @@ def check_stock_quantity_bulk(
         raise InsufficientStock(insufficient_stocks)
 
 
-def check_preorder_threshold_bulk(
+def get_channel_data(
```

Comment:
Would consider a more precise name for this function; get_channel_data can mean a lot of things. If this is used only in this module then we could also prefix it with an underscore to notify that it is something internal.

Reply:
Done, now function name I connected with type returned.